### ISSUE TITLE
hi, similar to my other image note it seems this one isn't working as intended either. This listing has a group image and individual images assigned to each variant. When I load the page, it shows the highlighted single at the top and the group undernea...

### ISSUE THREAD

**Issue by liamcottrell1** [Venture-Stream/aquacadabra-2.0#95]
hi, similar to my other image note it seems this one isn't working as intended either. This listing has a group image and individual images assigned to each variant. When I load the page, it shows the highlighted single at the top and the group underneath it. When I use the dropdown to select a different option, the main display image changes but the thumbnail doesn't change and then if you click between the group thumbnail and the top one, it then won't change the display to the one in the thumbnail I guess because another dropdown item is highlighted.

Anyway, in short it doesn't seem to be working as intended! Give me a shout if I can clarify further on my ramblings, cheers

[https://www.bugherd.com/projects/396350/tasks/59](https://www.bugherd.com/projects/396350/tasks/59)

**Comment by liamcottrell1** [comment]
still broken unless I'm missing something. happy to jump on a call if it might help?

it's not ideal to not have an image for every variant but we need it to work with/without them please

**Comment by liamcottrell1** [comment]
@[Ray Woods](495048) I've just deployed a fix for this, would you be able to clear your cache/view the site in a private tab and check again for me please?
